util/comparator.cc

```diff
@@ -7,13 +7,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file. See the AUTHORS file for names of contributors.
 
+#include "rocksdb/comparator.h"
```

Comment:
Re-order the headers?

Reply:
This is done by our automatic linter.

I believe the linter follows the same order as suggested by Google C++ Style, and I believe it's the same as what our linter made: https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

This file is comparator.cc, so it ranked comparator.h to the first.